src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -329,7 +329,16 @@ const PublicationReleasePage: NextPage<Props> = ({ data }) => {
       <PublicationReleaseHeadlinesSection release={data} />
 
       {data.content.length > 0 && (
-        <Accordion id="content">
+        <Accordion
+          id="content"
+          onSectionOpen={accordionSection => {
+            logEvent(
+              `${data.publication.title} release page`,
+              `${data.publication.title} accordion opened`,
+              `${accordionSection.title} section opened`,
```

Comment:
Actually no need to repeat ourselves. Think we can just leave the label as the `accordionSection.title` by itself